fmriprep/workflows/bold/confounds.py

```diff
@@ -1080,10 +1084,12 @@ def _carpet_parcellation(segmentation, crown_mask, nifti=False):
     lut[100:201] = 1 if nifti else 0  # Ctx GM
     lut[30:99] = 2 if nifti else 0    # dGM
     lut[1:11] = 3 if nifti else 1     # WM+CSF
-    lut[255] = 4 if nifti else 0      # Cerebellum
+    lut[255] = 5 if nifti else 0      # Cerebellum
     # Apply lookup table
     seg = lut[np.asanyarray(img.dataobj, dtype="uint16")]
-    seg[np.asanyarray(nb.load(crown_mask).dataobj, dtype=int) > 0] = 5 if nifti else 2
+    seg[np.asanyarray(nb.load(crown_mask).dataobj, dtype=int) > 0] = 6 if nifti else 2
+    # Separate deep from shallow WM+CSF
+    seg[np.asanyarray(nb.load(acompcor_mask).dataobj, dtype=int) > 0] = 4 if nifti else 1
```

Comment:
This is currently converting what are almost certainly `uint8`s into `int64` before converting to `bool` with `> 0`. Let's just convert directly.

```suggestion
    seg[np.bool(nb.load(crown_mask).dataobj)] = 6 if nifti else 2
    # Separate deep from shallow WM+CSF
    seg[np.bool(nb.load(acompcor_mask).dataobj)] = 4 if nifti else 1
```